### PR TITLE
Working on notes

### PR DESCRIPTION
**Pull request by DKaramfilov**

### CODE REVIEW

**Review by velikovmartin — CHANGES_REQUESTED** [review]

File: controls/radspreadsheet/features/notes.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Notes
+page_title: Notes 
+slug: radspreadprocessing-features-notes
```
[the remark this answers is not in the document]
Comment (DKaramfilov) [reply controls/radspreadsheet/features/notes.md:4]: fixed

File: controls/radspreadsheet/features/notes.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+tags: notes, xlsx
+published: True
+position: 12
+---
+
+# Notes
+
+Since R1 2022 the Spreadsheet control supports working with notes. The Notes are used for making notes or annotations about the data. All notes can be found in the __NoteCollection__ of the worksheet. More information about using notes in code is available here: [SpreadProcessing Notes](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/notes).
```
[the remark this answers is not in the document]
Comment (DKaramfilov) [reply controls/radspreadsheet/features/notes.md:12]: fixed

File: controls/radspreadsheet/features/notes.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+As you can see in the above image, the notes menu allows you to perform the following actions as well:
+
+* __EditNote:__ Puts the note of the current cell in edit mode.
+* __Select Previous or Next Note:__ Selects a note depending on the currently selected cell. 
+* __Show/Hide Note:__ Shows or hides the note of the current cell. 
+* __Show/Hide All Notes:__ Shows or hides all the notes in the current sheet. When enabled all notes will be visible and you will be able to select and edit them. 
+
+### Editing Notes
+
+To edit a note select a cell that contains note and click "Edit Note" from the context menu.
```
[the remark this answers is not in the document]
Comment (DKaramfilov) [reply controls/radspreadsheet/features/notes.md:39]: fixed

File: controls/radspreadsheet/features/notes.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Figure 4: Edit Note
+
+![RadSpreadProcessing Notes](images/RadSpreadProcessing_Notes_04.png)
+
+### Deleting Notes
+
+To delete a note select it an click "Delete Note" from the context menu.
+
```
[the remark this answers is not in the document]
Comment (DKaramfilov) [reply controls/radspreadsheet/features/notes.md:48]: added

**Review by velikovmartin — APPROVED** [review]

**Pull request merged**